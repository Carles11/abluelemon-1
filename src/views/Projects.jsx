import React from 'react'
import Helmet from 'react-helmet'

const Projects = () => (
  <section>
    <Helmet 
      title="Projects" 
      meta={[
        { name: "description", content: "Projects" },
      ]}
    />
  Projects
  </section>
)
  


export default Projects