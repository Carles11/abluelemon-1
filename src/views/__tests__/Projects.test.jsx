import React from 'react'
import { shallow } from 'enzyme'
import Projects from '../Projects'

describe('<Projects />', () => {
  it('should render itself', () => {
    const component = shallow(<Projects />)
    expect(component).toMatchSnapshot()
  })
}) 