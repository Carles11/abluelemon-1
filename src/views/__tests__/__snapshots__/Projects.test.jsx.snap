// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Projects /> should render itself 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Projects />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
          meta={
            Array [
              Object {
                "content": "Projects",
                "name": "description",
              },
            ]
          }
          title="Projects"
        />,
        "Projects",
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "defer": true,
          "encodeSpecialCharacters": true,
          "meta": Array [
            Object {
              "content": "Projects",
              "name": "description",
            },
          ],
          "title": "Projects",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "Projects",
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
            meta={
              Array [
                Object {
                  "content": "Projects",
                  "name": "description",
                },
              ]
            }
            title="Projects"
          />,
          "Projects",
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "defer": true,
            "encodeSpecialCharacters": true,
            "meta": Array [
              Object {
                "content": "Projects",
                "name": "description",
              },
            ],
            "title": "Projects",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "Projects",
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
